supertokens_python/recipe/emailverification/asyncio/__init__.py

```diff
@@ -37,20 +55,73 @@ async def verify_email_using_token(
 
 
 async def is_email_verified(
-    user_id: str, email: str, user_context: Union[None, Dict[str, Any]] = None
+    user_id: str,
+    email: Optional[str] = None,
+    user_context: Union[None, Dict[str, Any]] = None,
 ):
     if user_context is None:
         user_context = {}
-    return await EmailVerificationRecipe.get_instance().recipe_implementation.is_email_verified(
+
+    recipe = EmailVerificationRecipe.get_instance()
+    if email is None:
+        email_info = await recipe.get_email_for_user_id(user_id, user_context)
+        if isinstance(email_info, GetEmailForUserIdOkResult):
+            email = email_info.email
+        elif isinstance(email_info, EmailDoesnotExistError):
+            return True
+        else:
+            raise Exception("Unknown User ID provided without email")
+
+    return await recipe.recipe_implementation.is_email_verified(
+        user_id, email, user_context
+    )
+
+
+async def revoke_email_verification_token(
+    user_id: str,
+    email: Optional[str] = None,
+    user_context: Optional[Dict[str, Any]] = None,
+):
+    if user_context is None:
+        user_context = {}
+
+    recipe = EmailVerificationRecipe.get_instance()
+    if email is None:
+        email_info = await recipe.get_email_for_user_id(user_id, user_context)
+        if isinstance(email_info, GetEmailForUserIdOkResult):
+            email = email_info.email
+        elif isinstance(email_info, EmailDoesnotExistError):
+            # Here we are returning OK since that's how it used to work, but a later call
+            # to is_verified will still return true
+            return CreateEmailVerificationTokenEmailAlreadyVerifiedError()
```

Comment:
this should return `RevokeEmailVerificationTokensOkResult`. This is also a mistake in the node PR, and I have commented in that.

Reply:
Also the comment you added vs the code you wrote.. copy / pasting error?